pilot/pkg/xds/ads.go

```diff
@@ -147,11 +147,15 @@ func (s *DiscoveryServer) receive(con *Connection, identities []string) {
 		req, err := con.stream.Recv()
 		if err != nil {
 			if istiogrpc.IsExpectedGRPCError(err) {
-				log.Infof("ADS: %q %s terminated", con.peerAddr, con.conID)
+				if req != nil && req.TypeUrl != v3.SecretType {
```

Comment:
Why do you check not secret type?

Reply:
Because for secret types we do not want to log this messages as it can be 1000s as it logs for every secret.

Reply:
np, without ratelimit, will  error happen

Reply:
This is actually important for new connection with out this check 

2022-09-29T03:33:23.515201Z	info	ads	ADS: new connection for node:istio-ingressgateway-processing-us-west-2c-6c74f945bc-6cpvz.sfproxy-13


lots of these mis leading logs with come. So I protected at both places

Reply:
This can happen only at L180, how about changing to debug

Reply:
I think that log at info level is useful

Reply:
Seems like we are hiding a useful error message. This seems to be because we are mixing "SecretType" to mean "istio-agent SDS server", when its not the case...